config/config.go

```diff
@@ -56,7 +56,7 @@ var (
 		CloudAPIToken:         newCfg("cloud.api.token", ""),
 		Context:               newCfg("context", ""),
 		Contexts:              newCfg("contexts", ""),
-		DockerCommand:         newCfg("container.binary", "docker"),
+		DockerCommand:         newCfg("container.binary", ""),
```

Comment:
`docker` is no longer the "default" here. We "default" to an empty string and search the `$PATH` for an appropriate binary. If one is not found, we now throw a more helpful error.